### DISCUSSION TITLE
Volunteer PR reviewers wanted

### DISCUSSION THREAD

**Discussion by SethTisue** [slick/slick#2196] [General]
There are a number of open pull requests at https://github.com/slick/slick/pulls needing reviewers

I've tweeted about it here, retweets appreciated: https://twitter.com/SethTisue/status/1376599766751084547

**Comment by nafg** [comment]
BTW, I wonder if it would be easier, if instead of a `needs review` label, if we could configure the repo so all PRs require a review. PRs can be filtered by review required. Then we can just use the github Review function to "approve" PRs.

**Reply by SethTisue** [reply]
yes, I would be fine with deleting that label, and I'm also fine with making the configuration change you suggest

**Reply by nafg** [reply]
I think only you can change the repo settings

**Reply by nafg** [reply]
You can filter either with the Review dropdown on the PRs list page or going straight to https://github.com/slick/slick/pulls?q=is%3Apr+is%3Aopen+review%3Arequired

**Reply by SethTisue** [reply]
I've deleted the label and enabled branch protection on all branches, as follows:

<img width="1136" alt="Screen Shot 2021-04-12 at 10 09 19 AM" src="https://user-images.githubusercontent.com/161079/114433919-3f478f80-9b77-11eb-93df-050f55973da7.png">

happy to adjust further, as needed

**Reply by SethTisue** [reply]
(...CI is now re-running on every open PR, so that'll take a while to clear...)
